### PR TITLE
Adding Database Keep Export Tests

### PR DESCRIPTION
**Pull request by ahsu1230**
Changed the subtitle under Export section

@martinraison @eishay

### CODE REVIEW

File: kifi-backend/modules/shoebox/test/com/keepit/controllers/website/KeepsControllerTest.scala [hunk 1]
```diff
@@ -760,7 +760,63 @@ class KeepsControllerTest extends Specification with ApplicationInjector {
...
 
-    "keepExports" in  {
+    "export keeps" in {
+      running(new ShoeboxApplication(controllerTestModules: _*)) {
+        val t1 = new DateTime(2014, 7, 4, 21, 59, 0, 0, DEFAULT_DATE_TIME_ZONE)
+        val userRepo = inject[UserRepo]
+        val uriRepo = inject[NormalizedURIRepo]
+        val urlRepo = inject[URLRepo]
+        val keepRepo = inject[KeepRepo]
+        val keeper = KeepSource.keeper
```
Comment (martinraison) [inline kifi-backend/modules/shoebox/test/com/keepit/controllers/website/KeepsControllerTest.scala:770]: looks like your not actually using it

File: kifi-backend/modules/shoebox/test/com/keepit/controllers/website/KeepsControllerTest.scala [hunk 2]
```diff
@@ -760,7 +760,63 @@ class KeepsControllerTest extends Specification with ApplicationInjector {
...
+        val initLoad = KeepSource.bookmarkImport
```
Comment (martinraison) [inline kifi-backend/modules/shoebox/test/com/keepit/controllers/website/KeepsControllerTest.scala:771]: same here

File: kifi-backend/modules/shoebox/test/com/keepit/controllers/website/KeepsControllerTest.scala [hunk 3]
```diff
@@ -760,7 +760,63 @@ class KeepsControllerTest extends Specification with ApplicationInjector {
...
+        val db = inject[Database]
+        val ColRepo = inject[CollectionRepo]
+        val Keep2ColRepo = inject[KeepToCollectionRepo]
```
Comment (martinraison) [inline kifi-backend/modules/shoebox/test/com/keepit/controllers/website/KeepsControllerTest.scala:774]: we typically use camelCase for variable names (first letter is lower case)

**Comment by martinraison** [comment]
Great! Feel free to merge after addressing the comments

**Comment by ahsu1230** [comment]
Please look at new pull request #9260 . kinda messed up by branching or something

**Pull request closed without merging**